### PR TITLE
feat: new library AWS Architecture Icons

### PR DESCRIPTION
**Pull request by excalibot**
AWS Architecture Icons

Submitted by @ChildishGirl

Items: EMR, Kinesis, Kinesis Data Firehose, Kinesis Video Streams, Kinesis Data Analytics, Kinesis Data Streams, CloudSearch, Athena, Glue, Crawler, DataCatalog, Elastic Views, Lake Formation, Redshift, QuickSight, Data Pipeline, Managed Streaming for Apache Kafka, Data Exchange, Glue DataBrew, OpenSearch Service, SNS, SQS, Amazon MQ, AppSync, Step Functions, EventBridge, Managed Workflows for Apache Airflow, AppFlow, API Gateway, Endpoint, SES, Alexa for Business, Chime, Budgets, Cost & Usage Report, Cost Explorer, EC2, Elastic Beanstalk, Lambda, Outposts, EC2 Image Builder, App Runner, Batch, Connect, EKS, ECR, ECS, Container, Fargate, Support, Aurora, DocumentDB, DynamoDB, Keyspaces, DMS, Neptune, Quantum Ledger Database, RDS, Timestream, ElastiCache, MemoryDB for Redis, Cloud9, CodeBuild, CodeCommit, CodeDeploy, CodePipeline, X-Ray, CodeArtifact, Amplify, Device Farm, Location Service, IoT Analytics, IoT Core, IoT ExpressLink, IoT TwinMaker, IoT FleetWise, IoT Device Defender, IoT Device Management, IoT Events, IoT RoboRunner, IoT Greengrass, IoT SiteWise, Comprehend, Elastic Inference, Forecast, Lex, Personalize, Polly, Rekognition, SageMaker, Notebook, Canvas, Model, Train, Ground Truth, Textract, Transcribe, Translate, Deep Learning AMIs, Deep Learning Containers, DeepLens, DeepComposer, Augmented AI, Fraud Detector, Kendra, DevOps Guru, Panorama, SageMaker Studio Lab, Comprehend Medical, CloudTrail, CloudWatch, Alarm, Dashboard, Logs, System Manager, Chatbot, Service Catalog, Application Auto Scaling, Config, Personal Health Dashboard, CloudFormation, Well-Architected Tool, Control Tower, Backint Agent, Distro for Open Telemetry, Fault Injection Simulator, Managed Service for Grafana, Managed Service for Prometheus, OpsWorks, System Manager Incident Manager, Resilence Hub, Organizations, AppConfig, Transfer Family, Application Discovery Service, DataSync, Migration Hub, Server Migration Service, Application Migration Service, Migration Evaluator, Transit Gateway, CloudFront, Route53, VPC, Peering connection, Internet gateway, NAT gateway, VPN gateway, NACL, ELB, ALB, NLB, Direct Connect, PrivateLink, Client VPN, Site-to-Site VPN, Global Accelerator, Network Firewall, Cognito, GuardDuty, Inspector, Macie, Artifact, Certificate Manager, CloudHSM, Directory Service, Firewall Manager, IAM, Single Sign-On, KMS, Secrets Manager, Security Hub, Shield, WAF, Detective, Resource Access Manager, Audit Manager, EBS, EFS, Storage Gateway, S3 Bucket, FSx, Backup, AWS account, Region, Auto Scaling group, AWS Step Functions Workflow

[Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fchildishgirl-aws-architecture-icons-1679492061652%2Flibraries%2Fchildishgirl%2Faws-architecture-icons.excalidrawlib%3Fraw%3Dtrue)

### CODE REVIEW

**Comment by ChildishGirl** [comment]
Hi @dwelle, it's a major update for an existing AWS Architecture icons library and I wanted to ask if it's possible to replace previous version with this one? I added some changes to already existing icons and made proper ordering for all icons, so users can navigate through them quicker. 
Also, I don't know if Excalidraw has any rules on libraries preview images, but so many icons (187) will be too small to see them in one preview image. I can prepare preview image with 10-20 icons to give users some idea of icons style if it is allowed. What do you think?

**Comment by kkrav3ts** [comment]
@dwelle could you please merge this pack? I am fostering using this tool in my organization, but this update is really needed because introduces massive amount of changes. Is there any chance this can be added soon?

@ChildishGirl Is there any way how I can use them before this is merged?

**Comment by ChildishGirl** [comment]
@kkrav3ts I think you can use [Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fchildishgirl-aws-architecture-icons-1679492061652%2Flibraries%2Fchildishgirl%2Faws-architecture-icons.excalidrawlib%3Fraw%3Dtrue) to use them before they are merged.

@dwelle could you please let me know if I can do something to speed up the process?

File: libraries.json [hunk 1]
```diff
@@ -3444,82 +3444,208 @@
...
         "twitter": "",
-        "url": "https://www.linkedin.com/in/annpastushko/"
+        "url": "https://www.linkedin.com/in/annpastushko"
       }
     ],
     "source": "childishgirl/aws-architecture-icons.excalidrawlib",
     "preview": "childishgirl/aws-architecture-icons.jpg",
     "created": "2022-08-26",
-    "updated": "2022-12-27",
+    "updated": "2023-03-22",
```
Comment (dwelle) [inline libraries.json:3459]:
```suggestion
    "updated": "2023-04-27",
```

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
Hey @ChildishGirl — too many notifications lately, and some of them get forgotten :(. Merging, thanks! ❤️

**Pull request merged**